module/index.ts

```diff
@@ -135,10 +139,10 @@ export class Client {
       const req = protocol.request(url, {
         method: 'POST',
         headers: {...headers,},
-      }, (res) => {
+      }, (res: http.IncomingMessage) => {
         let body = '';
 
-        res.on('data', (chunk) => {
+        res.on('data', (chunk: any) => {
```

Comment:
is the type always a string? can we type it as string here?

Reply:
Yeah I think that makes sense here.